tutorials/matchmaker101/frontend/main.go

```diff
@@ -63,17 +63,19 @@ func main() {
 // once it has an assignment.
 func deleteOnAssign(fe pb.FrontendServiceClient, t *pb.Ticket) {
 	for {
+		time.Sleep(time.Second * 1)
```

Comment:
Reason for adding this delay?

Reply:
sorry, I think this is added accidentally.